src/librustc/hir/lowering.rs

```diff
@@ -1131,12 +1131,12 @@ impl<'a> LoweringContext<'a> {
 
     fn lower_token(&mut self, token: Token, span: Span) -> TokenStream {
         match token {
-            Token::Interpolated(_) => {}
+            Token::Interpolated(nt) => {
+                let tts = Token::interpolated_to_tokenstream(&self.sess.parse_sess, nt, span);
+                self.lower_token_stream(tts)
+            }
             other => return TokenTree::Token(span, other).into(),
```

Comment:
The return here should probably also be dropped.

Reply:
Good catch. I've fixed it.